test/check-machines-nics

```diff
@@ -908,6 +908,163 @@ class TestMachinesNICs(machineslib.VirtualMachinesCase):
                     self.browser.wait_not_present(f"#vm-subVmTest1-network-{self.nic_num}-mac")
                 self.browser.wait_not_present(".pf-v6-c-modal-box")
 
+    @testlib.skipImage('PASST not supported', "rhel-8-*", "ubuntu-2204", "ubuntu-2404")
+    def testPortForwards(self):
+        b = self.browser
+
+        # Ubuntu produces AppArmor denials...
+        self.allow_journal_messages('.*apparmor="DENIED".*operation="sendmsg".*comm="passt.avx2".*')
+
+        self.login_and_go("/machines", superuser=False)
+        self.waitPageInit()
+
+        self.createVm("subVmTest1", connection="session", running=False)
+        self.goToVmPage("subVmTest1", "session")
+
+        # Machine should have a NIC with "default" backend.
+
+        self.expandNicRow(1)
+        b.wait_in_text("#vm-subVmTest1-network-1-type", "user")
+        b.wait_in_text("#vm-subVmTest1-network-1-backend", "default")
+
+        # Add port forwards for 8080-8090 -> 80 and 127.0.0.1 2222 -> 22
+
+        b.click("#vm-subVmTest1-network-1-edit-dialog")
+        b.wait_visible("#vm-subVmTest1-network-1-edit-dialog-modal-window")
+        b.click("#vm-subVmTest1-network-1-edit-dialog-port-forwards button:contains(Add)")
+        b.set_input_text("#vm-subVmTest1-network-1-edit-dialog-port-forwards-0-host-port", "8080-8090")
+        b.set_input_text("#vm-subVmTest1-network-1-edit-dialog-port-forwards-0-guest-port", "80")
+        b.click("#vm-subVmTest1-network-1-edit-dialog-port-forwards button:contains(Add)")
+        b.set_input_text("#vm-subVmTest1-network-1-edit-dialog-port-forwards-1-ip-address", "127.0.0.1")
+        b.set_input_text("#vm-subVmTest1-network-1-edit-dialog-port-forwards-1-host-port", "2222")
+        b.set_input_text("#vm-subVmTest1-network-1-edit-dialog-port-forwards-1-guest-port", "22")
+        b.click("#vm-subVmTest1-network-1-edit-dialog-save")
+        b.wait_not_present("#vm-subVmTest1-network-1-edit-dialog-modal-window")
```

Comment:
UDP is not covered at all in the tests, but maybe more important what if I want to forward both? This isn't something we actually handle in cockpit-podman you'll have to add both ranges manually which is imo fine.